### ISSUE TITLE
Skin not loading

### ISSUE THREAD

**Issue by BlizzBlu** [FTBTeam/FTB-Presents-Direwolf20-1.12#120]
* **Modpack Version**: v2.5.0

* **Issue**: Skin does not load every time I play the game (Singleplayer world) - it always defaults to the Alex skin

* **Link to Log or Crash File Paste**: If the Launcher Console suffices, here's the link: https://paste.feed-the-beast.com/view/99d57f84

* **Is it Repeatable?**: Yes, on every boot (it's unavoidable)

* **Mod/s Affected**: Unknown

* **Known Fix**: Unknown

**Comment by Sunekaer** [comment]
please reupload log, as it's not available anymore

**Comment by BlizzBlu** [comment]
I honestly forgot I opened this issue,

The problem was that ReAuth refused to connect to the servers, leaving me offline and thus without my skin.
I never would've figured this out if I didn't play multiplayer at least a few times, since I was forced to reconnect before I could play on any server, and ReAuth's button does not show up in the Singleplayer menu.
The issue has disappeared since then, as I saved my password and told ReAuth to log me in automatically upon bootup. It's not the most optimal solution, but it's something that works.
